icebergs_framework.F90

```diff
@@ -756,6 +767,269 @@ subroutine move_berg_between_cells(bergs)  !Move icebergs onto the correct lists
 
 end subroutine move_berg_between_cells
 
+
+! #############################################################################
+
+subroutine update_halo(bergs)
```

Comment:
Trivial thing, but can we call it update_halo_bergs() to make it clear the function is not doing a gridded data halo update?